src/main/java/com/yurjinia/user/service/UserProfileService.java

```diff
@@ -89,11 +91,28 @@ public void deleteAvatar(String userEmail) {
         userProfileMapper.toDto(userProfileEntity);
     }
 
+    public UserProfileDTO updateUserProfile(String userEmail, MultipartFile image, UpdateUserProfileRequest updateUserProfileRequest) {
+        UserProfileEntity userProfileEntity = getUserProfileByEmail(userEmail);
+
+        if (updateUserProfileRequest != null) {
```

Comment:
the whole if statement you can move to separate method and call - updateProfileMetaData